content/sensu-core/2.0/reference/checks.md

```diff
@@ -312,6 +312,20 @@ description  | If the event is to be silenced.
 type         | boolean
 example      | {{< highlight shell >}}"silenced": false{{< /highlight >}}
 
+|output_metric_format    |      |
+-------------|------
+description  | The metric format generated by the check command. Sensu supports the following metric formats: <br>`nagios_perfdata` ([Nagios Performance Data][nagios]) <br>`graphite_plaintext` ([Graphite Plaintext Protocol][graphite]) <br>`influxdb_line` ([InfluxDB Line Protocol][influx]) <br>`opentsdb_line` ([OpenTSDB Data Specification][open])<br><br>When a check includes an `output_metric_format`, Sensu will extract the metrics from the check output and add them to the [event data][24] within the `metrics` scope. For more information about extracting metrics using Sensu, see the [guide][23]. 
+required     | false (true if `output_metric_handlers` is configured)
+type         | String
+example      | {{< highlight shell >}}"output_metric_format": "graphite_plaintext"{{< /highlight >}}
+
+|output_metric_handlers    |      |
+-------------|------
+description  | The metric formats expected by the `handlers` specified in the check definition. Sensu supports the following metric handler formats: <br>`nagios_perfdata` ([Nagios Performance Data][nagios]) <br>`graphite_plaintext` ([Graphite Plaintext Protocol][graphite]) <br>`influxdb_line` ([InfluxDB Line Protocol][influx]) <br>`opentsdb_line` ([OpenTSDB Data Specification][open])<br><br>When a check includes one or more `output_metric_handlers`, Sensu will transform metrics from the format specified by the `output_metric_format` attribute into the formats specified by `output_metric_handlers` before adding the metrics to the event data. For more information about extracting metrics using Sensu, see the [guide][23]. 
```

Comment:
I think the description for the 4 metric formats is sufficient in `output_metric_formats`. `output_metric_handlers` are actually not confined to just those metric formats, since a metric handler can send a Sensu metric _anywhere_. For example, I believe Prometheus has a time series database that a handler could send these metrics to! `output_metric_handlers` are simply a list of [Sensu handlers](https://github.com/sensu/sensu-docs/blob/master/content/sensu-core/2.0/reference/handlers.md) specifically designed to handler event data containing metrics.

Reply:
Yes thats the intention! [Pipelined treats all handlers the same](https://github.com/sensu/sensu-go/blob/91df85824a5f521798e24d9ed6746dfd9e04c93c/backend/pipelined/handle.go#L47-L53) regardless though, but it's definitely best practice to reserve `output_metric_handlers` for handlers related to metrics.